src/test/java/uk/gov/hmcts/ccd/domain/types/OrgPolicyCaseAssignedRoleValidatorTest.java

```diff
@@ -0,0 +1,135 @@
+package uk.gov.hmcts.ccd.domain.types;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import uk.gov.hmcts.ccd.data.caseaccess.CaseRoleRepository;
+import uk.gov.hmcts.ccd.data.definition.CaseDefinitionRepository;
+import uk.gov.hmcts.ccd.domain.model.definition.CaseFieldDefinition;
+import uk.gov.hmcts.ccd.domain.model.definition.CaseTypeDefinition;
+import uk.gov.hmcts.ccd.test.CaseFieldDefinitionBuilder;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasSize;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+@DisplayName("OrgPolicyCaseAssignedRoleValidator")
+class OrgPolicyCaseAssignedRoleValidatorTest {
+
+    private static final JsonNodeFactory NODE_FACTORY = JsonNodeFactory.instance;
+    private static final String FIELD_ID = "TEST_FIELD_ID";
+
+    @Mock
+    private CaseDefinitionRepository definitionRepository;
+
+    @Mock
+    private CaseRoleRepository caseRoleRepository;
+
+    private OrgPolicyCaseAssignedRoleValidator validator;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.initMocks(this);
+
+        when(definitionRepository.getBaseTypes()).thenReturn(Collections.emptyList());
+        BaseType.setCaseDefinitionRepository(definitionRepository);
+        BaseType.initialise();
+        validator = new OrgPolicyCaseAssignedRoleValidator(caseRoleRepository);
+    }
+
+
+    @Test
+    @DisplayName("should fail due to incorrect organisation role")
+    void failDueToIncorrectOrganisationRole() {
+
+        final CaseFieldDefinition caseFieldDefinition = caseField().build();
+        caseFieldDefinition.setId("TEST");
+        final JsonNode validValue = NODE_FACTORY.textNode("XXXX");
+        ValidationContext validationContext1 = createValidationContext(caseFieldDefinition, validValue);
+        final List<ValidationResult> validResult = validator.validate(validationContext1);
+
+        assertAll(
+            () -> assertThat("Expected input to be valid", validResult, hasSize(1)),
+            () -> assertTrue(validResult.get(0).getErrorMessage()
+                .contains("The value XXXX is not a valid organisation role."))
```

Comment:
Can we make this and all other tests currently using `contains` check for equality instead? Looks like we can always know what the values should be so should be able to check for an exact match.

Reply:
yeap , I agree ☝️